root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -1736,6 +1736,13 @@ const {LINK_TYPES, cleanURL, guessType, validationRules} = require('../../edit/U
                      input_entity_type: 'release',
             expected_relationship_type: 'discographyentry',
         },
+        // Ko-fi
+        {
+                             input_url: 'http://ko-fi.com/35MJZ8OL4IO',
+                     input_entity_type: 'artist',
+            expected_relationship_type: 'patronage',
+                    expected_clean_url: 'https://www.ko-fi.com/35MJZ8OL4IO',
```

Comment:
You should have removed `ẁww` from the output/expected clean URL rather than from the input URL.

Reply:
Ahh I think I forgot to compile my tests when checking.